tests/rules/property.js

```diff
@@ -111,7 +111,19 @@ eslintTester.run("property", rule, {
         {
             code: "w.innerHTML = `<span>${'lulz'+'meh'}</span>`;",
             parserOptions: { ecmaVersion: 6 }
-        }
+        },
+
+        // testing unwrapSafeHTML spread
+        {
+            code: "this.imeList.innerHTML = Sanitizer.unwrapSafeHTML(...listHtml);",
+            parserOptions: { ecmaVersion: 6 }
+        },
+
+
+        // Native method (Check customize code doesn't include these)
+        {
+            code: "document.toString = evil;"
```

Comment:
So user defines $ruleChecks to be {x: {}}

and both are considered invalid if not checked correctly in our customization code:
```
v.x = "blah";
```

```
b.toString = z;
```


This is mostly for `toString()` method check but I added it for parity.